src/main/webapp/app/shared/participant-scores/participant-scores.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input } from '@angular/core';
+import { ParticipantScoreDTO } from 'app/shared/participant-scores/participant-scores.service';
+
+@Component({
+    selector: 'jhi-participant-scores',
+    templateUrl: './participant-scores.component.html',
+    styles: [],
```

Comment:
No style is applied, this can be removed.

Reply:
done